src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  &__title {
+    padding-inline: 16px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+
+    position: absolute;
+    top: 206px;
+    transition-duration: $trans-duration;
```

Comment:
The variable `$trans-duration` is used but not defined in this file. Ensure that it is defined elsewhere in your stylesheets or consider defining it here.